src/dosarray_run_http_experiment.sh

```diff
@@ -10,6 +10,19 @@ then
   exit 2
 fi
 
+while getopts ":g" opt; do
+  case ${opt} in
+    g )
+      GRAPHING=true
+      ;;
+    ? )
+      echo "Usage: ./dosarray_run_http_experiment [-g] <server-choice> <attack-choice>"
```

Comment:
It would be more accurate to write "target-choice" rather than "server-choice" since the target might not be a server (e.g., it might be a cache).